src/collect-forms.rst

```diff
@@ -17,7 +17,7 @@ In order to fill out forms with survey participants, you must first load blank f
 
 .. _in-app-get-blank-forms:
 
-Loading Forms from a server
+Loading forms from a server
```

Comment:
One quick thought: should we actually always be capitalizing ODK concepts (forms, projects, instances etc) to highlight that we're not just using the normal word? This is something that seems to be really inconsistently done in the docs, but we could just add a decision about it (arbitrary or not) to the style guide?

Reply:
Sigh. Yes, we should capitalize and be consistent.